docs.json

```diff
@@ -9,6 +9,12 @@
     [
       "Another Page",
       "/another-page"
+    ],
+    [
+      "Weiteres",
+      [
+        ["Alpha-Programm", "/more/alpha-program"]
+      ]
```

Comment:
My idea is to have for every feature (like homework, timetable, events, etc.) a own section and a section "more" where we have non feature things (like the alpha program). Later, we might going to split the "more" section if it becomes to big.

Reply:
Hm, okay, the only thing is that the URL might not be stable then if we split off the article later to some other section.  
Okay with me though, since we don't have an existing, better way to categorize the articles.

Reply:
Yes, I also don't know to fix that really. I also wanted to create a shorter link for some things, like our alpha program: https://sharezone.net/alpha which would redirect to the docs. At least we could update there the link. However, I think our traffic for the docs will not that much high, so I think it's okay.